WordPress/Helpers/MinimumWPVersionTrait.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Helpers;
+
+use PHP_CodeSniffer\Files\File;
+use PHPCSUtils\BackCompat\Helper;
+
+/**
+ * Helper utilities for sniffs which take the minimum supported WP version of the
+ * code under examination into account.
+ *
+ * Usage instructions:
+ * - Add a trait `use` statement to this trait in the class which intends to use this functionality.
```

Comment:
This sentence reads a little odd to me:

```
Add a trait `use` statement to this trait...
```

Shouldn't this be 

```
Add a trait `use` statement in the class which intends to use this functionality.
```

?

Reply:
It should probably have read "referring to this trait" or something, but I agree, the text can/should be made clearer.

In reality, we'd normally want:
- An import `use` statement to the trait in the file using it
- A trait `use` statement using the unqualified name in the class using it

Now, how to say that in a clear mar, while still being concise and not presuming that everyone codes like we do...

Reply:
I'm fine with having this broken down into two separate points. I like to think that people who will use this will know how to import traits into their classes 👀

Reply:
> I like to think that people who will use this will know how to import traits into their classes

That's exactly why I tried to address it in one sentence... having it broken down into the minutia feels like talking down to people.

Reply:
In that case, I think just saying that one should use it should suffice. The actual how-to will be known to the person using it.

Reply:
Updated now to "Add appropriate `use` statement(s) to the file/class which intends to use this functionality." Does that work for you ?